### ISSUE TITLE
Support custom emojis

### ISSUE THREAD

**Issue by twouters** [Sorunome/mx-puppet-slack#33]
Custom emojis are frequently used on my workspace, so I'm missing out on the fun :(

During initial connection we can get a full list of current emojis: https://api.slack.com/methods/emoji.list

When new emojis are added or old ones are changed we get an emoji_changed event on the event api.

I guess this should wait until there's official custom emoji support in matrix?

**Comment by Sorunome** [comment]
> I guess this should wait until there's official custom emoji support in matrix?

nah, mx-puppet-discord already supports custom emojis. Didn't know slack had custom emojis until now! How do you create/use them within slack, so that soru can play around and debug and stuffs?

**Comment by twouters** [comment]
Dunno, I don't use Slack :roll_eyes: 

I'll have a look.

**Comment by twouters** [comment]
If you click the emoji in the input bar (bottom right) you can add emoji's
Can't find any api calls for it though :(

**Comment by Sorunome** [comment]
Soru meant how to do it in the UI, thanks!
